templates/tweet/tweet-detail.html

```diff
@@ -0,0 +1,15 @@
+{% extends "includes/base.html" %}
+{% block main_content %}
+  <a href="{% url 'home' %}" style="float: right;">Home</a>
+  <h4>Tweets Details</h4>
+  <div class="post">
+      <h6>{{ tweet_details.user.first_name }}</h6>
+      <p>{{ tweet_details.tweet_message }}</p>
```

Comment:
improper indentations

Reply:
fixed. I did with prettier but that make some django code in single line so I just correct that code. I'll make sure this never occurs in future